ops/docker/extract_build_args.jq

```diff
@@ -0,0 +1,8 @@
+def compute_build_args($input; $container_id):
```

Comment:
Could you please share an example input and example output of this script? Nice little language, but difficult to visualize like `sed` to newbies like me.

Reply:
I will add it as part of the docstring.